pkg/storage/stores/shipper/downloads/table_manager.go

```diff
@@ -309,9 +321,11 @@ func (tm *TableManager) ensureQueryReadiness(ctx context.Context) error {
 			return err
 		}
 
+		perTableStart := time.Now()
 		if err := table.EnsureQueryReadiness(ctx, usersToBeQueryReadyFor); err != nil {
 			return err
 		}
+		level.Debug(util_log.Logger).Log("msg", "instance query ready for users", "users", usersToBeQueryReadyFor, "query_readiness_duration", time.Since(perTableStart), "table", tableName)
```

Comment:
```suggestion
		level.Info(util_log.Logger).Log("msg", "index pre-download for query readiness completed", "users", usersToBeQueryReadyFor, "duration", time.Since(perTableStart), "table", tableName)
```